common/shared/src/test/java/app/coronawarn/server/common/shared/util/HashUtilsTest.java

```diff
@@ -0,0 +1,32 @@
+package app.coronawarn.server.common.shared.util;
+
+import static app.coronawarn.server.common.shared.util.HashUtils.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import app.coronawarn.server.common.shared.util.HashUtils.MessageDigestAlgorithms;
+import com.google.protobuf.ByteString;
+import org.junit.jupiter.api.Test;
+import java.security.NoSuchAlgorithmException;
+
+class HashUtilsTest {
+
+  public static final String TEST_STRING = "string";
+
+  @Test
+  void testMd5DigestAsHex() {
+    String hash = md5DigestAsHex(TEST_STRING);
+    assertThat(hash).hasSize(32);
+  }
+
+  @Test
+  void testRandomByteArrayData() {
+    byte[] hash = generateSecureRandomByteArrayData(16);
+    assertThat(hash).hasSize(16);
```

Comment:
Maybe we can also refactor this test. The advantage of the secure random is that even if we use the same seed for two different random variables the result will be different